### ISSUE TITLE
Tikz and tabu (negativ width coeﬃcients) conflict

### ISSUE THREAD

**Issue by maboloshi** [tabu-fixed/tabu#23]
### MWE
```latex
% -*- coding: utf-8 -*-
%!TEX program = xelatex
\documentclass[12pt]{article}
\usepackage{tabu}
\usepackage{tikz}
\begin{document}

\begin{table}[htb]
    \centering
    \begin{tabu}to.87\textwidth{*3{X[c]|}X[-2.5,c]*2{|X[c]}} 
        XX & X    & X    & X        & X    & X    \\
           & XX & XX & XXXXXX & XX & XX \\
    \end{tabu}
\end{table}
\end{document}
```

### LOG
> This is XeTeX, Version 3.14159265-2.6-0.999992 (MiKTeX 2.9.7440 64-bit)
```latex
! Missing } inserted.
<inserted text> 
                }
l.13     \end{tabu}
                   
```

**Comment by u-fischer** [comment]
tikz loads xcolor. This activates additional grouping commands in various places to avoid that colors leak out and tabu can't cope with this. Your issue is basically the same as issue #1

**Comment by maboloshi** [comment]
After testing, it is found that the version of tabu in MiKTeX is 2.9. Automatic update cannot obtain the new version. After manually updating to 2.10, the problem is solved.

**Comment by mrpiggi** [comment]
Is there any reason, why v2.10 wasn't shipped to CTAN?

**Comment by u-fischer** [comment]
@mrpiggi if I remember correctly it is still broken in other places. Also as the readme says: there is no maintainer. Do you want to take over the package? Or do you know someone who wants it?

**Comment by davidcarlisle** [comment]
@mrpiggi  2.10 fixed a couple of the reported issues but two or three _new_ issues were reported  on 2.10 before it had even gone to ctan, at which point we decided we simply didn't have the cycles to maintain this.

**Comment by mrpiggi** [comment]
@davidcarlisle That is completely understandable.

@u-fischer 
> Do you want to take over the package? 

Absolutely no way! The source code is a mess. Taking over maintenance would mean to re-develop the package from scratch, probably with a complete new user-interface and provide some backward compatibility. But I have no time to implement this. I already opened a feature request https://github.com/latex3/latex2e/issues/176 and I saw https://github.com/latex3/latex2e/projects/7 as well. I am fully aware that the LaTeX project team has a lot to do and this is not something that can be done in a short time.

> Or do you know someone who wants it?

Unfortunately not.
